### DISCUSSION TITLE
Cannot Pull Keda Images from github project url

### DISCUSSION THREAD

**Discussion by piyapan039285** [kedacore/keda#3261] [General] [answered]
I cannot pull images from keda project. here is my command output.

```
docker pull ghcr.io/kedacore/keda:2.6.1

2.6.1: Pulling from kedacore/keda
2df365faf0e3: Pulling fs layer
0f20782e98cc: Pulling fs layer
error pulling image configuration: download failed after attempts=1: unauthorized: unauthenticated: User cannot be authenticated with the token provided.
```

version 2.7.0, 2.7.1 also have the same problem. I also tried pulling image on Azure cloud server and have the same problem as well. 

Anyone have any recommendation on how to fix this?
Thanks.

**Comment by mhowell24** [comment, marked as answer]
I belive this might be because githubs container registry is having issues

**Reply by tomkerkhove** [reply]
Yes, that is correct: https://www.githubstatus.com/
